docs_src/microservices/application/services/AppServiceConfigurable/GettingStarted.md

```diff
@@ -0,0 +1,40 @@
+# App Service Configurable
+## Getting Started
+
+To get started with App Service Configurable, you'll want to start by determining which functions are required in your pipeline. Using a simple example, let's assume you wish to use the following functions from the SDK:
+
+1. [FilterByDeviceName](../details/AvailablePipelineFunctions/#filterbydevicename) -  to filter events for a specific device.
+2. [Transform](../details/AvailablePipelineFunctions/#transform) - to transform the data to XML
```

Comment:
Does Transform support only XML data conversion or any other formats ?

Reply:
XML and JSON.